src/HospitalLibrary/Appointments/AppointmentService.cs

```diff
@@ -27,6 +28,26 @@ public Task<IEnumerable<Appointment>> GetAll()
             return _unitOfWork.AppointmentRepository.GetAll();
         }
 
+        public async Task<IEnumerable<TimeInterval>> GetTimeIntervalsForStepByStep(int doctorId, DateTime chosen)
+        {
+            WorkingHours doctorsWorkingHours = _unitOfWork.WorkingHoursRepository.GetOne((int)chosen.DayOfWeek,doctorId);
```

Comment:
I think DayOfWeek is original data type, so why passing the value as int?

Reply:
DayOfWeek  is type itself not int, so cant be changed